src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -3,46 +3,154 @@
-    result = get_vm_sizes(location)
-    return [r.name for r in result]
+from ._completers import get_vm_size_completion_list
+from ._validators import validate_create_parameters
+from ._validators import validate_k8s_version
+from ._validators import validate_k8s_client_version
+from ._validators import validate_linux_host_name
+from ._validators import validate_list_of_integers
+from ._validators import validate_ssh_key
+
+
+def load_arguments(self, _):
+
+    # ACS command argument configuration
+    with self.argument_context('acs') as c:
+        c.argument('resource_name', name_type,
+                   completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'),
+                   help='Name of the container service. You can configure the default using `az configure --defaults acs=<name>`')
+        c.argument('name', name_type,
+                   completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'),
+                   help='Name of the container service. You can configure the default using `az configure --defaults acs=<name>`')
+        c.argument('container_service_name', name_type, help='Name of the container service. You can configure the default using `az configure --defaults acs=<name>`',
+                   completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'))
+        c.argument('admin_username', options_list=['--admin-username', '-u'], default='azureuser')
+        c.argument('api_version',
+                   help=_get_feature_in_preview_message() + 'Use API version of ACS to perform az acs operations. Available options: 2017-01-31, 2017-07-01. Default: the latest version for the location')
+        c.argument('dns_name_prefix', options_list=['--dns-prefix', '-d'])
+        c.argument('orchestrator_type', get_enum_type(orchestrator_types), options_list=['--orchestrator-type', '-t'])
+        c.argument('ssh_key_value', required=False, type=file_type, default=os.path.join('~', '.ssh', 'id_rsa.pub'),
+                   completer=FilesCompleter())
+        c.argument('tags', tags_type)
+        c.argument('disable_browser', help='Do not open browser after opening a proxy to the cluster web user interface')
+
+    with self.argument_context('acs create') as c:
+        c.argument('ssh_key_value', required=False, type=file_type, default=os.path.join('~', '.ssh', 'id_rsa.pub'),
+                   completer=FilesCompleter(), validator=validate_ssh_key)
+        c.argument('master_profile', options_list=['--master-profile', '-m'], type=validate_file_or_dict,
+                   help=_get_feature_in_preview_message() + 'The file or dictionary representation of the master profile. Note it will override any master settings once set')
+        c.argument('master_vm_size', completer=get_vm_size_completion_list, help=_get_feature_in_preview_message())
+        # c.argument('master_osdisk_size', type=int, )
```

Comment:
unfinished?

Reply:
Just some cruft I'll remove. `master_osdisk_size` is properly defined a couple lines below.